src/components/SearchBar/SearchBar.module.scss

```diff
@@ -6,32 +6,54 @@
 .searchBar input {
 	width: 100%;
 	height: 32px;
-	background: var(--color-cosmic-gradient-25);
+	background: none;
+	border: 1px solid var(--color-purple-lighter);
 	border-radius: 31px;
 	padding: 0 24px 1px 39px;
 
 	/* Text */
 	text-align: left;
-	color: white;
+	color: #bfbfbf;
 
 	/* Text shadow isn't working as expected - it cuts off for some reason */
 	/* text-shadow: var(--glow-text-white); */
 }
 
-.searchBar button {
+.searchBar .ButtonIconStyles {
 	position: absolute;
 	padding: 0;
 	left: 16px;
 	top: 9px;
 	height: 14px;
 	width: 14px;
+	background-image: url('./assets/search-icon-gray.svg');
 	background-color: rgba(0, 0, 0, 0);
-	background-image: url('./assets/search-icon.svg');
 	background-position: center;
 	background-size: 14px 14px;
 	background-repeat: no-repeat;
 }
 
+.searchBar:focus-within .ButtonIconStyles {
+	background-image: url('./assets/search-icon.svg');
+}
+
 .searchBar input::placeholder {
-	color: white;
+	color: #bfbfbf;
+}
+
+.searchBar input:hover {
+	// background: #1d154a;
+	background: var(--color-background-alt-50)
+}
+.searchBar input:focus::placeholder {
+	color: #fff;
+}
+.searchBar input:focus {
+	color: #fff;
+	background: linear-gradient(
+		90deg,
+		rgba(137, 48, 238, 0.25) 0%,
+		rgba(175, 114, 243, 0.25) 100%
+	);
+
 }
```

Comment:
Remember we're working in `.scss` files now, so you can simplify this to:

```scss
.searchBar {
    input {

        &:hover {}

        &:focus {
            &::placeholder {}
        }


    }
}
```

Not a major drama, just letting you know so we can transition properly to SASS over time

Reply:
Done. Thank you, I think this is the right way to do it.

Reply:
Done. I think this is the right way to do it.